docs/docsite/rst/dev_guide/developing_modules_checklist.rst

```diff
@@ -24,6 +24,7 @@ To contribute a module to Ansible, you must:
 * minimize module dependencies
 * support :ref:`check_mode <check_mode_dry>` if possible
 * ensure your code is readable
+* if a module is named `<something>_facts`, it is because it is returning `ansible_facts`, do not name modules that do not do this with `_facts`.
```

Comment:
```suggestion
* if a module is named `<something>_facts`, it is because it is returning `ansible_facts`. Do not name modules that do not do this with `_facts`.
```